index.bs

```diff
@@ -2981,9 +2981,65 @@ Wikipedia also describes some <a href="https://en.wikipedia.org/wiki/Monkey_patc
+
+    <div class="example" id="example-monkeypatch-quote-text-before-step">
+
+    Before step 4.2 of [=HTTP fetch=], "If request's redirect mode is "follow", then set request's service-workers mode to "none".", insert the following steps:
+    1. The first new step.
+
+    </div>
+    <div class="example" id="example-monkeypatch-quote-text-replace-def">
+
+    In terms of CSS2.1 block-level formatting [[CSS2]], the rules for “over-constrained” computations in [[CSS2#blockwidth]] are ignored in favor of alignment as specified here and the used value of the margin properties are therefore not adjusted to correct for the over-constraint.
+
+    </div>
+    <div class="example" id="example-monkeypatch-quote-text-change-idl">
+
+    Append `optional USVString value` parameters to the definitions of <code>{{URLSearchParams}}.{{URLSearchParams/delete()}}</code> and <code>{{URLSearchParams}}.{{URLSearchParams/has()}}</code>:
+
+    <pre class="idl">
+    partial interface URLSearchParams {
+      undefined delete(USVString name<ins>, optional USVString value</ins>);
+      boolean has(USVString name<ins>, optional USVString value</ins>);
+    };
+    </pre>
+
+    </div>
+    <div class="example" id="example-monkeypatch-quote-text-change-css">
+
+    Modify the definition of <a spec="css-fonts-4" property>font-size-adjust</a> as follows:
+
+    <pre class="propdef">
+    Name: font-size-adjust
+    Value: none | <ins>[ ex-height | cap-height | ch-width | ic-width | ic-height ]? [ from-font |</ins> <<number>> <ins>]</ins>
+    Initial: none
+    Applies to: all elements and text
+    Inherited: yes
+    Percentages: N/A
+    Computed value: <del>a number or the keyword none</del><ins>the keyword ''font-size-adjust/none'', or a pair of a metric keyword and a <<number>></ins>
+    Animation type: <ins>discrete if the keywords differ, otherwise </ins>by computed value type
+    </pre>
+
+    </div>
+
+ 1. Keep monkey patches short. If you're modifying an algorithm by adding more than a couple of steps, define a separate, self-contained algorithm in your specification, and have the monkey patch call the new algorithm.
+ 1. If you're replacing or adding steps in an algorithm, write the new steps so an editor can paste them verbatim into the upstream algorithm. This implies that control flow like "return" or "abort" will return or abort from the whole upstream algorithm, not just the monkey patched section.
+ 1. File an issue against the existing specification that asks to merge your monkey patch. This will notify the relevant community that there's a change incoming that might benefit from their review. This community may suggest better ways to accomplish your goals. Take them seriously. They might also be able to tell you how to hook into an existing extension point or quickly create one so that you can remove your monkey patch sooner than you expected.
```

Comment:
This doesn’t really apply if the specification is at a very early stage (e.g. stage 1 TC39 proposals)